omero-web/README.md

```diff
@@ -0,0 +1,33 @@
+OMERO.web
+=========
+
+Docker image for testing OMERO.web on CentOS 7 with systemd.
+
+Alter omero config in omeroweb.config as needed
+
+1. Build:
+
+        docker build -t omeroweb-deploy .
+
+    with custom zip:
+
+        docker build -t omeroweb-deploy --build-arg ZIPURL=https://path.to.omeropy.zip .
+
+
+2. Run on Linux:
```

Comment:
It might be easier to refer to the README for https://github.com/ome/ome-docker/tree/master/omero-ssh-systemd since it varies.